block/config.ml

```diff
@@ -1,6 +1,26 @@
 open Mirage
 
-let main = foreign "Unikernel.Main" (console @-> block @-> job)
+type shellconfig = ShellConfig
+let shellconfig = Type ShellConfig
+
+let config_shell = impl @@ object
+    inherit base_configurable
+
+    method configure i =
+      let dir = Info.root i in
+      Functoria_app.Cmd.run "dd if=/dev/zero of=%s/disk.img count=100000" dir
+
+    method clean i =
+      let dir = Info.root i in
+      Functoria_app.Cmd.run "rm -f %s/disk.img" dir
```

Comment:
@Drup would be really nice if functoria provided a safe (array-based) alternative to `run`. String-interpolation + `rm -f` is not a good example to set... (will break if `dir` contains spaces or other special characters, etc).